libs/dotnet-sdk/project.json

```diff
@@ -22,7 +22,22 @@
         },
         "lint": {
             "executor": "@nx-dotnet/core:format"
+        },
+        "package": {
+            "executor": "@nx-dotnet/core:build",
+            "outputs": [
+                "{workspaceRoot}/dist/libs/dotnet-sdk"
+            ],
+            "options": {
+                "configuration": "Release",
```

Comment:
Would you want default to be Debug? At the moment `--configuration production` does nothing (which is fine).

Reply:
No, when creating the nuget package I believe it should be release. I remove the configuration production which came from the build step above.